assets/queries/k8s/service_with_external_load_balance/query.rego

```diff
@@ -5,30 +5,35 @@ CxPolicy[result] {
 	service.kind == "Service"
 	metadata := service.metadata
 
-	not CheckIfDestinationPortExists(service)
+    service.spec.type == "LoadBalancer"
+    service.spec.selector.app
+
+	documents := input.document
+	pod := [pod | documents[index].spec.template.metadata.labels.app == service.spec.selector.app; pod = documents[index]]
+	count(pod) != 0
+	checkLabels(pod, service.spec.selector.app)
+	res := checkPorts(pod, service.spec.ports[k].targetPort)
+	res == true
```

Comment:
Could only be `checkPorts(pod, service.spec.ports[k].targetPort)` and refactor to returns true or false